### PR TITLE
added r-ca

### PR DESCRIPTION
**Pull request by MathiasHaudgaard**

### CODE REVIEW

File: recipes/r-ca/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('ca')"  # [not win]
+    - "\"%R%\" -e \"library('ca')\""  # [win]
+
+about:
+  home: http
```
Comment (bgruening) [inline recipes/r-ca/meta.yaml]: url is missing

File: recipes/r-ca/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+test:
+  commands:
+    - $R -e "library('ca')"  # [not win]
+    - "\"%R%\" -e \"library('ca')\""  # [win]
+
+about:
+  home: https://cran.r-project.org/web/packages/ca/index.html
+  license: GPL
+  summary: Computation and visualization of simple, multiple and joint correspondence analysis.
+  license_family: GPL
```
Comment (bgruening) [inline recipes/r-ca/meta.yaml]: license_file is missing here

**Comment by bgruening** [comment]
@MathiasHaudgaard thanks a bunch! i think I merged all recipes that have passed the build besides this one. The rest are broken in some way or the other and needs manually tweaking. Let me know if you need help with them.

**Comment by MathiasHaudgaard** [comment]
Thanks @bgruening 😊 
I have some questions related to the failing packages.
Some packages fail to be built for x64. Should I remove the PR for those packages or can they be fixed in one way or another?

Other packages fail on travis but I'm not quite sure why. Maybe they just need a rebuild? :shipit:

File: recipes/r-ca/meta.yaml (hunk 2 above)
Comment (MathiasHaudgaard) [reply recipes/r-ca/meta.yaml]: Hmm, don't know what to do about this one. It's now included when creating the recipe through conda skeleton cran 😕

**Comment by bgruening** [comment]
Ideally they should be fixed and they probably can. Please add a note to those PRs so we can look over them.

The travis once maybe just need a restart I will try to get to those.
Thanks for all your work here and have a look at: https://github.com/bioconda/bioconda-recipes/issues/7498

File: recipes/r-ca/meta.yaml (hunk 2 above)
Comment (bgruening) [reply recipes/r-ca/meta.yaml]: you can just add this one-liner with `license_file:`?

Comment (jdblischak) [reply recipes/r-ca/meta.yaml]: @MathiasHaudgaard Here's how I was advised to handle the license for R packages that are ambiguous about the GPL version:

https://github.com/conda-forge/staged-recipes/pull/4676/files#diff-ab236354f787c30256ad43a827c41f18R42

```
about:
  license: GPL-2.0 and GPL-3.0
  license_family: GPL
  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win]
```

**Pull request merged**